src/core/layout/gel-button.js

```diff
@@ -114,6 +109,11 @@ export class GelButton extends Phaser.GameObjects.Container {
         );
     }
 
+    setClickSound(key) {
+        this.config.clickSound = key;
+        this._click = this.scene.sound.add(key);
```

Comment:
is there a test for this being called and `play()` being called on it on pause event?

Reply:
Oh good point - there's marginal testing through the contructor but since we can call this directly now I'll add some one ta :)